src/create.js

```diff
@@ -73,14 +77,26 @@ module.exports = function createDriftwood (primaryLogger) {
     globalState.loggers.push(log)
     return log
 
+    function intercept (args) {
+      if (interceptors && interceptors.length > 0) {
+        for (var i = 0; i < interceptors.length; i++) {
+          args = interceptors[i](args.slice())
```

Comment:
Can we add a safety check here: if an interceptor returns something falsey, we shouldn't overwrite the args. Means you can use an interceptor as an observer without having to remember to return the same args